pkg/test/parameter.go

```diff
@@ -7,23 +7,52 @@ package test
 
 import (
 	"bytes"
+	"encoding/json"
 	"errors"
 	"fmt"
 	"text/template"
 
 	"github.com/facebookincubator/contest/pkg/target"
 )
 
-// Param represents a test step parameter. It is initialized from a string,
-// which can be either a host name, a function expression, or just any regular
-// string. It enables hostname and function expression validation and a few
-// other validation operations. This is meant to simplify the way plugin access
-// and use test step parameters.
-type Param string
+// NewParam inititalizes a new Param object directly from a string.
+// Note that no validation is performed if the input is actually valid JSON.
+// This helper is primarily intended for tests.
```

Comment:
since it's only for tests, this should be private

Reply:
nevermind, I see that this is used by  `sshcmd`. Maybe we should not hint that this is mainly for tests?

Reply:
Hmm yeah I did not think about that when I wrote the commit, I might just remove that remark.

Alternatively, I can change `sshcmd`, it does not have to create a `Param` there, it might just a string. But I guess it is more consistent if we keep it `Param`.

Reply:
I removed the comment